toolchain/semantics/semantics_handle_call_expression.cpp

```diff
@@ -41,8 +42,13 @@ auto SemanticsHandleCallExpression(SemanticsContext& context,
                                          /*diagnostic=*/nullptr));
 
   // TODO: Propagate return types from callable.
+  SemanticsTypeId type_id = callable.return_type_id;
+  // For functions with no return type, setting return type to empty tuple type.
```

Comment:
```suggestion
  // For functions with an implicit return type, set the return type to empty tuple type.
```

Having an implicit return type is distinct from having no return type, so I want to be specific about that in comments.